DataConnectors/WorkplaceFacebook/WorkplaceFacebook_Webhooks_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+{
+	"id": "WorkplaceFacebook",
+	"title": "Workplace from Facebook",
+	"publisher": "Facebook",
+	"descriptionMarkdown": "The [Workplace](https://www.workplace.com/) data connector provides the capability to ingest common Workplace events into Azure Sentinel through Webhooks. Webhooks enable custom integration apps to subscribe to events in Workplace and receive updates in real time. When a change occurs in Workplace, an HTTPS POST request with event information is sent to a callback data connector URL.  Refer to [Webhooks documentation](https://developers.facebook.com/docs/workplace/reference/webhooks) for more information. The connector provides ability to get events which helps to examine potential security risks, analyze your team's use of collaboration, diagnose configuration problems and more.",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. ​Follow the steps to use this Kusto functions alias **Workplace_Facebook** in queries and workbooks [Follow steps to get this Kusto functions>](https://aka.ms/sentinel-WorkplaceFacebook-parser).",
+	"graphQueries": [{
+			"metricName": "Total data received",
+			"legend": "Workplace_Facebook_CL",
+			"baseQuery": "Workplace_Facebook_CL"
+		}
+	],
+	"sampleQueries": [{
+		"description": "Workplace Events - All Activities.",
+		"query": "Workplace_Facebook\n | sort by TimeGenerated desc"
+	}
+],
+	"dataTypes": [{
+			"name": "Workplace_Facebook_CL",
+			"lastDataReceivedQuery": "Workplace_Facebook_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [{
+			"type": "IsConnectedQuery",
+			"value": [
+				"Workplace_Facebook_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		}
+	],
+	"availability": {
+		"status": 2,
```

Comment:
Status should be 1. Could you please update the status.

Thanks,
Aditi Maheshwari

Reply:
fixed